utils/build-presets.ini

```diff
@@ -1839,6 +1839,10 @@ install-swift
 install-llbuild
 install-swiftpm
 install-swiftsyntax
+swift-testing
+swift-testing-macro
```

Comment:
```suggestion
swift-testing-macros
```

Reply:
Nice catch. this could have been the cause of a failure I was seeing in https://github.com/swiftlang/swift-package-manager/pull/8092